src/main/resources/data/quark/tags/hedges.json

```diff
@@ -0,0 +1,5 @@
+{
+  "replace": false,
+  "values": [
+  ]
+}
```

Comment:
What is the purpose of this file? Nothing is added to the tag.

Reply:
I think it's supposed to be referenced in `HedgeBlock#canConnect` instead of the instanceof check, so that it can connect with Quark hedges

Reply:
Yeah, that was the original intention but it looks like I reverted that decision sometime later incorrectly. Thanks for fixing it up Bagel 👍